envconnect/templates/envconnect/improve.html

```diff
@@ -45,7 +45,7 @@
                             [[practice[0].consumption.nb_respondents]] respondents
                             <div class="progress" style="margin:0">
                                 <div class="progress-bar progress-bar-success green-level-[[practice[0].consumption.avg_value]]" role="progressbar" aria-valuenow="[[practice[0].consumption.rate]]" aria-valuemin="0" aria-valuemax="100" ng-style="implementationRateWidth(practice)">
-                                    [[practice[0].consumption.rate|number:0]]%
+                                    <span ng-class="{'text-muted': practice[0].consumption.rate === 0}">[[practice[0].consumption.rate|number:0]]%</span>
```

Comment:
We might want to use `practice[0].consumption.rate < 1` because `[[practice[0].consumption.rate|number:0]]` will round any floating point value from 0 to 1 to "0". Still OK to merge. Thanks.